onnxruntime/contrib_ops/cuda/bert/fast_gelu_impl.cu

```diff
@@ -97,6 +97,46 @@ bool LaunchFastGeluKernel(const cudaDeviceProp& prop, cudaStream_t stream, int i
   return CUDA_CALL(cudaPeekAtLastError());
 }
 
+#if CUDA_VERSION >= 11000 && (__CUDA_ARCH__ >= 800 || !defined(__CUDA_ARCH__))
+template <unsigned TPB>
+__global__ void FastGeluKernel2(const nv_bfloat162 a, const nv_bfloat162 b, const nv_bfloat162 c,
+                                int input_length, int bias_length,
+                                const nv_bfloat162* input, const nv_bfloat162* bias, nv_bfloat162* output) {
+#if __CUDA_ARCH__ >= 530
```

Comment:
This may be undefined according to the above. Since this conditional covers the whole kernel, would it be more logical to merge it with the above #if?